src/constants/index.js

```diff
@@ -32,3 +32,15 @@ export const DRAFT_CONCEPT_ID_PREFIXES = {
   tool: 'TD',
   variable: 'VD'
 }
+
+/**
+ * Types of requests that can be send to EDL
+ */
+export const edlPathTypes = {
+  CREATE_GROUP: 'CREATE_GROUP',
```

Comment:
Discussed this on a call, this works together with `buildEdlGroupPath`, these constants are just providing a good dev experience with auto complete instead of typing out the strings